entrypoint.sh

```diff
@@ -119,9 +117,9 @@ DEBUG "Manifests: $MANIFESTS"
         # TODO: Check that DIGITALOCEAN_TOKEN SPACES_ACCESS_KEY_ID SPACES_SECRET_ACCESS_KEY are set
 
         # s3cmd DO remove bucket ENV VARIABLES
-        AWS_ACCESS_KEY_ID=${SPACES_ACCESS_KEY_ID}
-        AWS_SECRET_ACCESS_KEY=${SPACES_SECRET_ACCESS_KEY}
-        DIGITALOCEAN_TOKEN=${DIGITALOCEAN_TOKEN}
+        export AWS_ACCESS_KEY_ID=${SPACES_ACCESS_KEY_ID}
+        export AWS_SECRET_ACCESS_KEY=${SPACES_SECRET_ACCESS_KEY}
+        export DIGITALOCEAN_TOKEN=${DIGITALOCEAN_TOKEN}
```

Comment:
[p4] Why you need `export` here?

Reply:
The tool s3cmd that we use across functions requires it to be set that way

Reply:
[p4] Please add comments about this to code

Reply:
Done